api/rs/build/lib.rs

```diff
@@ -265,7 +218,11 @@ pub fn compile_with_config(
         }
     });
 
-    write!(code_formatter, "{}", generated).map_err(CompileError::SaveError)?;
+    let parsed: syn::File =
+        syn::parse2(generated).expect("internal error: invalid Rust code generated");
```

Comment:
should we really panic here? Or should we just leave the code unformatted?

I guess it's OK to panic because that would probably result in a compiler error later.
But that said it could just be a bug in syn.  And the error message might not be as good as the one from the rust compiler...

Reply:
Good point! I'll change it to unformatted as fallback.